sensor/kubernetes/complianceoperator/utils.go

```diff
@@ -38,6 +37,9 @@ func convertCentralRequestToScanSetting(namespace string, request *central.Apply
 		ObjectMeta: v1.ObjectMeta{
 			Name:      request.GetScanSettings().GetScanName(),
 			Namespace: namespace,
+			Labels: map[string]string{
```

Comment:
@rhmdnd It seems like this is probably a sufficient label.  Can you think of anything additional we may want?

Reply:
Not off the top of my head. I do have a general question on labels though.

If we decide to change the label in the future, or add an additional label, does sensor handle that migration and ensure older Compliance Operator resources are labeled appropriately moving forward?

Reply:
Not by default.  We would have to implement such.